ResearchKitTests/ORKRecorderTests.m

```diff
@@ -331,16 +333,18 @@ - (void)tearDown {
 
 - (void)recorder:(ORKRecorder *)recorder didCompleteWithResult:(ORKResult *)result {
      NSLog(@"didCompleteWithResult: %@", result);
+    _recorder = recorder;
     _result = result;
 }
 
 - (void)recorder:(ORKRecorder *)recorder didFailWithError:(NSError *)error {
     NSLog(@"didFailWithError: %@", error);
+    _recorder = nil;
     _result = nil;
 }
 
-- (ORKRecorder *)createRecorder:(ORKRecorderConfiguration *)conf {
-    ORKRecorder *recorder = [conf recorderForStep:[[ORKStep alloc] initWithIdentifier:@"step"]
+- (ORKRecorder *)createRecorder:(ORKRecorderConfiguration *)recorderConfiguration {
+    ORKRecorder *recorder = [recorderConfiguration recorderForStep:[[ORKStep alloc] initWithIdentifier:@"step"]
                                                                     outputDirectory:[NSURL fileURLWithPath:_outputPath]];
     recorder.delegate = self;
```

Comment:
Would be nice to add a test here to verify the identifier get passed from `ORKRecorderConfiguration` to `recorder`.